packages/p2p-v2/src/components/Modals/PreferredCountriesModal/PreferredCountriesHeader/PreferredCountriesHeader.tsx

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import { Text } from '@deriv-com/ui';
+
+const PreferredCountriesHeader = () => {
+    return <Text weight='bold'>Preferred countries</Text>;
+};
+
```

Comment:
does this need to be its own component?

Reply:
i just put it there since i did move out the other header components

Reply:
But its just a Text component, IMO it doesn't really need to be its own component because its a bit overkill. For sure its reusable but this can be the case for many Text components have have similar strings, meaning we should create one for each one? Usually you want to separate a component out if there is a lot of logic that could be managed solely by that component, and in this case its just returning a Text. 

Furthermore in terms of performance, each component has a cost in terms of performance, and react needs to keep track of each component's state, props, and lifecycle. So for a small piece of code liek this, the overhead of a component might outweigh the benefits.

Reply:
for a simple component like this, i dont think there is much overhead introduced with it and also it doesn't accept any state/props or there aren't any lifecycle methods associated with it.  Making it reusable also doesn't bring much of an advantage here, but smaller components look much neater. I think we can leave it as such.

Reply:
hmmm i still disagree tbh, we should directly just pass Text to the components, cause there are plenty of other Modals with full page modals that have the same header texts, but we dont need to create a new component for this. what do you guys think @farrah-deriv @jim-deriv @adrienne-deriv

Reply:
I agree, this is redundant and most likely not going to be reusable anywhere 🤣 

Too many modularization for components can lead to harder maintainability in the future 😅

Reply:
Performance is not a concern when splitting components into smaller ones, as `react` was built with exactly that in mind and large components can make code splitting hard, testability hard. The only reasonable issue would be @adrienne-deriv's concern, but then again, not splitting them up can lead to that as well. Bottom line is, single responsibility should be your aim, and like @farrah-deriv said, if something has meaning or purpose on it's own, then it should be split out.